src/databricks/labs/ucx/queries/progress/main/01_4_percentage_cluster_migration_readiness.sql

```diff
@@ -0,0 +1,5 @@
+/* --title 'Cluster migration readiness (%)' */
+SELECT
+    100 * COUNT_IF(size(failures) = 0) / COUNT(*) AS percentage
```

Comment:
```suggestion
    ROUND(100 * COUNT_IF(size(failures) = 0) / COUNT(*), 2) AS percentage
```

`73.91304347826087` -> `73.91`. probably `CEIL()` would be even better here.